jscomp/build_tests/super_errors/expected/repeated_def_types.res.expected

```diff
@@ -0,0 +1,11 @@
+
+  [1;31mWe've found a bug for you![0m
+  [36m/.../fixtures/repeated_def_types.res[0m:[2m3:6[0m
+
+  1 [2m│[0m type a
+  2 [2m│[0m 
+  [1;31m3[0m [2m│[0m type [1;31ma[0m
+  4 [2m│[0m 
+
+  Multiple definition of the type name a at line 1, characters 5-6.
```

Comment:
I think this should say 1:6 for consistency.

I think this is using ocaml locations, from `Location.`.
Instead it should be using super-errors locations. The super-error location treatment is use to output `3:6` above.
I think this should hook up in that location logic.

Reply:
Now I get it about error handling including super_error.
I'm going to fix this with super_error then will take care of #6202

Reply:
Looks like this test output has non changed yet?